### DISCUSSION TITLE
Nocodb setup project without UI

### DISCUSSION THREAD

**Discussion by ghost** [nocodb/nocodb#3585] [General]
When we create a new Nocodb container with docker,
is there any way to configure a default project connected to a bdd like with UI "Create by Connecting to an external database" but without to use UI.

**Comment by o1lab** [comment]
Not at the moment. Do you want to solely use it based on the API - Could you expand more ?

**Reply by dengmingcong** [reply]
Hello, is there a plan to support this feature, or is there a way to achieve this effect currently? 

Our production environment only needs to read data and will not write data, so we don't need a UI. We only need to read data through the API, which can save the time and resources required to start the UI.

**Reply by o1lab** [reply]
We dont have plans to for this i.e headless APIs or BaaS

**Comment by reitn** [comment]
I also want to this feature which can create project with external database using API.
